tests/AclMiddlewareTest.php

```diff
@@ -1,142 +1,154 @@
 <?php
 
-use App\API\V1\Entities\Role;
+use App\API\V1\Repositories\EmailVerificationRepository;
+use App\API\V1\Repositories\RoleRepository;
+use App\API\V1\Repositories\UserRepository;
 use Faker\Factory;
 use App\API\V1\Entities\User;
-use App\API\V1\Entities\EmailVerification;
 use TempestTools\Scribe\PHPUnit\CrudTestBaseAbstract;
 
 class AclMiddlewareTest extends CrudTestBaseAbstract
 {
-
-    /**
-     * @return User
-     */
-    protected function makeUser(): User
-    {
-        $generator = Factory::create();
-        $user = new User();
-        $user
-            ->setEmail($generator->safeEmail)
-            ->setPassword('password')
-            ->setName($generator->name)
-            ->setJob($generator->jobTitle)
-            ->setLocale('en')
-            ->setAddress($generator->address);
-        return $user;
-    }
-
+    protected $password = '441520435a0a2dac143af05b55f4b751';
```

Comment:
@jerazo if the password can be set as part of the env, then shouldn't be get the password here based on an encrypted version of the password in the env? Or am I not following how this works?

Reply:
Ok cool thanks